virt/virt-2-4-release-notes.adoc

```diff
@@ -24,6 +24,17 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 //CNV-5438 - svirt-support-for-cnv
 * This release features significant security enhancements. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged custom SCCs for VMs and limit the use of privileged SCCs to infrastructure containers in the cluster.
 
+//CNV-5472-Guest-OS
+* Supported guest operating system templates for {VirtProductName} include the following:
```

Comment:
I wonder if this should not be under `New and changed features` but rather a separate heading entirely, named something like `Supported guest operating systems`.

Then, under that heading, I'd write something like `{VirtProductName} guests can use the following operating systems:`

Reply:
Updated.